common/beerocks/scripts/prplmesh_utils.sh.in

```diff
@@ -39,6 +39,41 @@ killall_program() {
     done
 }
 
+is_wireless_iface() {
+        iwconfig "$1" > /dev/null 2>&1
+}
+
+is_bridge_iface() {
+    [ -d "/sys/class/net/$1/bridge" ] && return 0
+    return 1
+}
+
+platform_init() {
+    echo "platform init..."
+    base_mac=46:55:66:77
+    bridge_ip=192.168.250.140
+    
+    ip link add @BEEROCKS_BRIDGE_IFACE@ address "${base_mac}:00:00" type bridge
```

Comment:
You didn't mention in the commit message that you also add a stable MAC address to all the interfaces.

And obviously, that change should have been a separate commit too.